examples/labelbox.py

```diff
@@ -0,0 +1,8 @@
+from guizero import App, LabelBox, Text
+app = App()
+labelbox = LabelBox(app, "Label box")
+Text(labelbox,"Text inside a LabelBox")
+
+labelbox2 = LabelBox(app)
```

Comment:
Not sure of the benefit of a `LabelBox` without a label. How is this be different to a `Box`?

Reply:
Indeed the results are quite similar. A Box has a default behavior of not displaying a frame and if displayed, the frame color is customizable. The default behavior of a LabelBox is displaying a frame that will be similar in color to the background color of the LabeBox.